### DISCUSSION TITLE
2 color print

### DISCUSSION THREAD

**Discussion by Mario-vminion** [OrcaSlicer/OrcaSlicer#255] [Q&A] [answered]
i want to do a 2 color print on my minion , is there a manual or something that explain what the steps are to take in this slicer todo a 2 color print.

**Comment by hliebscher** [comment, marked as answer]
https://wiki.bambulab.com/en/x1/manual/multi-color-printing

https://www.youtube.com/watch?v=gl5TeluiHoY
